trust/test-extract.sh

```diff
@@ -97,4 +110,11 @@ test_blocklist()
 	assert_not_contains blocklist-test.pem $CERT_3_CN
 }
 
-run test_extract test_blocklist
+test_persist()
+{
+	trust dump --filter "pkcs11:object=$CERT_4_CN" > cert_4.p11-kit.dump
+	TD="cert_4.p11-kit.dump $TD"
+	assert_contains cert_4.p11-kit.dump nss-server-distrust-after:
```

Comment:
I believe this could be improved by checking for actual values as well and not just the presence of the field.